src/contexts/WalletContext/services.ts

```diff
@@ -52,6 +52,53 @@ export const checkForRiskyWallet = async (walletAddress: string) => {
   }
 };
 
+// Check if the provided signature is valid for the smart contract at signer.getAddress(),
```

Comment:
Thanks for the comments! Can we convert them to JS doc type of comments? Documenting the params of the function. You can find examples in the codebase on this. Look at src/utilities folder it has few nicely commented utils.

Reply:
Combined the two functions and added JS doc type comments.